src/main/java/pl/allegro/tech/embeddedelasticsearch/EmbeddedElastic.java

```diff
@@ -194,29 +194,38 @@ public void refreshIndices() {
 
     public static final class Builder {
 
-        private int portNumber = 9300;
-        private String clusterName = "elasticsearch";
+        private Map<String, Object> settings = new HashMap<>();
         private Optional<String> version = Optional.empty();
         private List<InstallationDescription.Plugin> plugins = new ArrayList<>();
         private Optional<URL> downloadUrl = Optional.empty();
         private Map<String, IndexSettings> indices = new HashMap<>();
 
         private Builder() {
+            this.settings.put("cluster.name", "elasticsearch");
```

Comment:
Istead of adding those defaults here, put them in InstanceDescription. Something like this:

```
    InstanceDescription(Map<String, Object> settings) {
        this.settings = ImmutableMap.<String, Object>builder()
                .put(TRANSPORT_TCP_PORT, DEFAULT_TRANSPORT_TCP_PORT)
                .put(CLUSTER_NAME, DEFAULT_CLUSTER_NAME)
                .putAll(settings)
                .build();
    }
```